charts/prometheus-thanos/templates/ruler-serviceaccount.yaml

```diff
@@ -1,8 +1,12 @@
-{{- if .Values.ruler.enabled -}}
+{{- if and .Values.ruler.enabled .Values.ruler.serviceAccount.create -}}
```

Comment:
I would omit the `ruler.serviceAccount.create` as the ruler always needs a service account because of the RBAC configuration. What do you think?

Reply:
It can be a good idea. If someone want to disable all service accounts, it can break the ruler. I change it !